### PR TITLE
Styling, placeholder pages

### PR DESCRIPTION
**Pull request by davidrideout**
* add loading of fonts
* add svg loader
* load custom theme json file colors
* add user flows for password reset and other pages (logic still needs to be implement)
* implement account page and updating user profile
* add styling to various existing pages.

### CODE REVIEW

**Comment by davidrideout** [comment]
Btw I added the swift file so my xcode would build -- not sure if will break yours. I can clean it up a little if necessary.

File: KukaApp/App.js [hunk 1]
```diff
@@ -46,21 +48,35 @@ export default () => {
...
+        customMapping={mapping}
+      >
         <SafeAreaProvider>
           <SafeAreaView
             style={{
               flex: 1,
               backgroundColor: theme['background-basic-color-2'],
             }}
           >
+            <StatusBar />
```
Comment (johnkmzhou) [inline KukaApp/App.js]: I don't believe this is necessary.

File: KukaApp/navigation/AuthNavigator.js [hunk 2]
```diff
@@ -36,6 +39,9 @@ export const AuthNavigator = () => {
   return (
     <Navigator tabBar={props => <BottomTabBar {...props} />}>
       <Screen name="Agenda" component={AgendaNavigator} />
+      <Screen name="Testimonies" component={TestimonyNavigator} />
+      <Screen name="My Account" component={MyAccountNavigator} />
```
Comment (johnkmzhou) [inline KukaApp/navigation/AuthNavigator.js:43]: Probably don't need navigators for some of these. That being said, we can leave it as is for now.

**Review by johnkmzhou — CHANGES_REQUESTED** [review]
Just a change request and a comment. Will post other thoughts in Slack channel.

**Review by johnkmzhou — APPROVED** [review]
lgtm

**Pull request merged**